graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -330,96 +330,112 @@ FusedCSCSamplingGraph::SampleNeighborsImpl(
   torch::optional<torch::Tensor> subgraph_type_per_edge = torch::nullopt;
 
   AT_DISPATCH_INTEGRAL_TYPES(
-      indptr_.scalar_type(), "SampleNeighborsImpl", ([&] {
-        const scalar_t* indptr_data = indptr_.data_ptr<scalar_t>();
-        auto num_picked_neighbors_data_ptr =
-            num_picked_neighbors_per_node.data_ptr<scalar_t>();
-        num_picked_neighbors_data_ptr[0] = 0;
+      indptr_.scalar_type(), "SampleNeighborsImplWrappedWithIndptr", ([&] {
```

Comment:
Don't change name?

Reply:
just to be consistent with following wrapper with `nodes`. Any concern about this name? any other dependency on this name beside its uniqueness requirement?